src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs

```diff
@@ -239,6 +239,23 @@ public async Task<SignConfigurationPdfResponse> GetPdfSignConfiguration(Guid sig
             }
         }
 
+        /// <summary>
+        /// Delete sign configuration.
+        /// </summary>
+        /// <param name="signConfigurationId">SignConfigurationID.</param>
+        /// <returns>HttpResponseMessage.</returns>
+        public async Task<HttpResponseMessage> DeletePdfSignConfiguration(Guid signConfigurationId)
+        {
+            var client = this.CreateClient();
+
+            using (var signConfigurationDetailsResponse = await client.DeleteSignConfigurationPdfWithHttpMessagesAsync(
+                 this.options.SubscriptionId,
+                 signConfigurationId).ConfigureAwait(false))
+            {
+                return signConfigurationDetailsResponse.Response;
```

Comment:
Can we add a switch case like we have in other methods?

Reply:
added switch case.